### PR TITLE
added Movie model

### PR DESCRIPTION
**Pull request by zulu607i**

### CODE REVIEW

File: movies/models.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    description = models.TextField(max_length=200, null=True, verbose_name='Description')
+    imdb_id = models.CharField(max_length=30, null=True, unique=True, verbose_name='IMDB_id')
+    length_min = models.IntegerField(verbose_name='Lenght')
+    is_scheduled = models.BooleanField(default=False)
+
+    def __str__(self):
+        return self.name
+
+    def get_imbd_url(self):
+        return f'{settings.IMDB_URL}{self.imdb_id}'
```
Comment (maxinasa) [inline movies/models.py:22]: See if this decorator might help: https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.functional.cached_property

**Comment by maxinasa** [comment]
I merged the 3rd PR, so this PR now has merge conflicts.
It was on purpose, just to see how you can handle this.

File: cinema/urls.py [hunk 2]
```diff
@@ -15,10 +15,13 @@
...
 from django.contrib import admin
 from django.urls import path, include
-
+from django.conf.urls.static import static
+from cinema import settings
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('ui.urls')),
     path('', include('users.urls')),
```
Comment (drejleeo) [inline cinema/urls.py:24]: aici include un prefix pentru fiecare aplicatie inclusa

**Review by drejleeo — COMMENTED** [review]
Take a look at what comments we left here

**Pull request merged**